lightbulb/client.py

```diff
@@ -1104,6 +1104,12 @@ async def handle_application_command_interaction(
 
         options, command = out
         context = self.build_command_context(interaction, options or [], command, initial_response_sent)
+        async with (
+                self.di.enter_context(di_.Contexts.DEFAULT),
+                self.di.enter_context(di_.Contexts.OPTION_CONVERSION) as container
+        ):
+            container.add_value(context_.Context, context)
+            await context.command._resolve_options()
```

Comment:
This shouldn't be done here, it should be part of the pipeline as discussed on discord